frontend/src/components/icons/PhSortDescending.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+	<svg xmlns="http://www.w3.org/2000/svg" width="1.2em" height="1.2em" viewBox="0 0 256 256">
```

Comment:
Auto-format all code using prettier

run `npx prettier . --write`
in your frontend folder

We will soon add it to pre-commit